src/locales/en_US.ts

```diff
@@ -352,6 +352,7 @@ const strings = {
   settings_exchange_settings: 'Exchange Settings',
   settings_exchange_instruction: 'Enable or disable the exchange providers available to you',
   settings_notifications_switch: 'Enable Price Alerts',
+  settings_spam_filter: 'Spam Transaction Filter',
```

Comment:
We should call the setting "Hide spam transactions". That makes it more clear what this feature does. We should also use sentence case, since most settings do that (with a small number of exceptions that might be worth fixing.)